scripts/build/VSImport-Samples.ps1

```diff
@@ -14,15 +15,18 @@ $ErrorActionPreference = "Stop"
 
 & $PSScriptRoot\unzip.ps1 -zipFile $zipFile -outputDir $tempDir
 
-$XCodeProjects = gci -recurse $tempDir\*.xcodeproj
+$XCodeProjects = gci -recurse $PSScriptRoot\..\..\samples\*.xcodeproj
+$XCodeProjects += gci -recurse $PSScriptRoot\..\..\tests\testapps\*.xcodeproj
```

Comment:
Why? Then we'd have to run the script twice and so we'd unzip tools twice. At that point we should just be running vsimporter directly and not using this script at all. 

---
In reply to: [103060288](https://github.com/Microsoft/WinObjC/pull/2097#discussion_r103060288) [](ancestors = 103060288)